### PR TITLE
feat: add -I option to cwctl and modify -i option

### LINKED ISSUE #4982: chatwoot ctl improvements
- [x] Fix non-zero exit on `cwctl --logs` option
- [x] fix exit handler when no options given
- [ ] --install should install master branch 
- [ ] Add -I to support installing custom branch

### PR DESCRIPTION
**Pull request by vishnu-narayanan**
# Pull Request Template

## Description

Modify `--install` option to drop support for installing specified branch. `-i` will always default to installing the latest stable branch(master).

Add `-I` option to support installing Chatwoot with a custom branch

Fixes #4982 

Related: https://github.com/chatwoot/chatwoot/pull/5037#issuecomment-1183831350

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: deployment/setup_20.04.sh [hunk 1]
```diff
@@ -615,8 +620,9 @@ Example: cwctl --upgrade
 Example: cwctl -c
 
 Installation/Upgrade:
-  -i, --install             install Chatwoot with the git branch specified
-  -u, --upgrade             upgrade Chatwoot to latest version
+  -i, --install             install the latest stable version of Chatwoot
```
Comment (pranavrajs) [inline deployment/setup_20.04.sh:623]:
```suggestion
  -i, --install             Install the latest stable version of Chatwoot
```

File: deployment/setup_20.04.sh [hunk 2]
```diff
@@ -615,8 +620,9 @@ Example: cwctl --upgrade
...
+  -I, --Install             install Chatwoot with the specified git branch
```
Comment (pranavrajs) [inline deployment/setup_20.04.sh:624]:
```suggestion
  -I, --Install             Install Chatwoot from a git branch
```

File: deployment/setup_20.04.sh [hunk 3]
```diff
@@ -615,8 +620,9 @@ Example: cwctl --upgrade
...
+  -u, --upgrade             upgrade Chatwoot to latest stable version
```
Comment (pranavrajs) [inline deployment/setup_20.04.sh:625]:
```suggestion
  -u, --upgrade             Upgrade Chatwoot to the latest stable version
```

File: deployment/setup_20.04.sh [hunk 4]
```diff
@@ -615,8 +620,9 @@ Example: cwctl --upgrade
...
   -s, --ssl                 fetch and install ssl certificates using LetsEncrypt
   -w, --webserver           install and configure Nginx webserver with SSL
```
Comment (pranavrajs) [inline deployment/setup_20.04.sh:627]:
```suggestion
  -w, --webserver           Install and configure Nginx webserver with SSL
```

File: deployment/setup_20.04.sh [hunk 5]
```diff
@@ -615,8 +620,9 @@ Example: cwctl --upgrade
 Example: cwctl -c
 
 Installation/Upgrade:
-  -i, --install             install Chatwoot with the git branch specified
-  -u, --upgrade             upgrade Chatwoot to latest version
+  -i, --install             install the latest stable version of Chatwoot
+  -I, --Install             install Chatwoot with the specified git branch
+  -u, --upgrade             upgrade Chatwoot to latest stable version
   -s, --ssl                 fetch and install ssl certificates using LetsEncrypt
```
Comment (pranavrajs) [inline deployment/setup_20.04.sh:626]:
```suggestion
  -s, --ssl                 Fetch and install SSL certificates using LetsEncrypt
```

**Review by pranavrajs — COMMENTED** [review]
Minor text changes. Otherwise let us get this merged in the release.

**Comment by vishnu-narayanan** [comment]
done

**Pull request merged**